corehq/apps/app_manager/views/translations.py

```diff
@@ -35,22 +35,34 @@ def get_index_for_defaults(langs):
 @require_can_edit_apps
 @get_file("bulk_upload_file")
 def upload_bulk_ui_translations(request, domain, app_id):
+
+    def _html_message(header_text, messages):
+        message = _(header_text or "Upload failed. We found problem with following translations:")
+        message += "<br>"
+        for prop in messages:
+            message += "<li>%s</li>" % prop
+        return message
+
     success = False
     try:
         app = get_app(domain, app_id)
-        trans_dict, error_properties = process_ui_translation_upload(
+        trans_dict, error_properties, warnings = process_ui_translation_upload(
             app, request.file
         )
         if error_properties:
-            message = _("We found problem with following translations:")
-            message += "<br>"
-            for prop in error_properties:
-                message += "<li>%s</li>" % prop
+            message = _html_message(error_properties)
             messages.error(request, message, extra_tags='html')
         else:
+            # update translations only if there were no errors
             app.translations = dict(trans_dict)
             app.save()
             success = True
+
+        if warnings:
+            message = _html_message("Upload succeeded, but we found following issues for some properties",
```

Comment:
you need translation markup here (even if it's being translated again) so that the translation engine picks up the string